packages/sui-decorators/src/decorators/cache/index.js

```diff
@@ -33,6 +33,13 @@ const _cache = ({
   const cache = caches[fnName]
 
   return (...args) => {
+    if (
+      (typeof window !== 'undefined' && window.__SUI_CACHE_DISABLED__) ||
+      global.__SUI_CACHE_DISABLED__
```

Comment:
shouldn't you check if `global` exists?

Reply:
Oh, I thought it would exist if `window` didn't !